### DISCUSSION TITLE
Bond orders

### DISCUSSION THREAD

**Discussion by abakan-zz** [prody/ProDy#1876] [Archived Issues]
Extend AtomGroup to store bond orders. `setBonds` and `addBonds` (to be implemented) should accept list of indices with bond orders (bonds.shape one of (n_bonds, 2) or (n_bonds, 3) with bond order). Also, implement `Bond.getOrder` method to be complete.

**Comment by jamesmkrieger** [comment]
setBonds exists

**Comment by jamesmkrieger** [comment]
I don't know if we need to do anything else about this but probably we do. There is a somewhat related issue (#798) and I did not find a `getBonds` method.

**Comment by jamesmkrieger** [comment]
This is handled by #1978
